src/modules/loadTodoListCard.js

```diff
@@ -0,0 +1,61 @@
+import savingData from './savingData.js';
+import updateCompleted from './updateCompleted.js';
+import reorder from './reorderArrayBasedOnIndex.js';
+
+export default function loadTodoListCard(arrayTodoList) {
+  function loopTodoElement(item) {
+    return `<li id="${item.index}" class="wrapper task-item" >
+                <span class="list ${item.completed === false ? '' : 'checked'}"><input class="checkbox" type="checkbox" data-id="${item.index}" ${item.completed === false ? '' : 'checked'} />${item.description}</span>
+                <span class="grab" draggable="true"><i class="icon fas fa-grip-vertical"></i></span>
+            </li>`;
+  }
+
+  const listContainer = document.querySelector('.list-container');
+
+  listContainer.innerHTML = arrayTodoList.map((list) => loopTodoElement(list)).join('');
+
+  const taskList = document.querySelectorAll('.task-item');
+  taskList.forEach((task) => {
+    task.addEventListener('dragstart', () => {
+      setTimeout(() => {
+        task.classList.add('dragging');
+      }, 0);
+    });
+
+    task.addEventListener('dragend', () => {
+      task.classList.remove('dragging');
+      const taskList = document.querySelectorAll('.task-item');
+      const index = [];
+      taskList.forEach((task) => {
+        index.push(Number(task.id) - 1);
+      });
+      reorder(arrayTodoList, index, arrayTodoList.length);
```

Comment:
- [Optional] I couldn't reorder the last two items on the list. I got the error below:
![Screenshot from 2023-06-05 09-28-29](https://github.com/ichsansandy/webpack-todo-list-microverse/assets/63087491/a840133b-b8b2-4efd-8fa7-b3ec8855a503)